academy/4-my-own-chain/play-game.md

```diff
@@ -174,6 +175,78 @@ Take the following steps to replace the `TODO`:
 
 That is all there is to it: good preparation and the use of Starport.
 
+## Interact via the CLI
+
+With one game in storage and the game waiting for `Bob`'s move, can `Alice` make a move? Check the order of the parameters:
+
+```sh
+$ checkersd tx checkers play-move --help
+Broadcast message playMove
+
+Usage:
+  checkersd tx checkers play-move [idValue] [fromX] [fromY] [toX] [toY] [flags]
+...
+```
+
+So `Alice` tries with:
+
+```sh
+$ checkersd tx checkers play-move 0 0 5 1 4 --from `echo $alice`
+                                     ^ ^ ^ ^ ^
+                                     | | | | To Y
+                                     | | | To X
+                                     | | From Y
+                                     | From X
+                                     Game id
+...
+raw_log: 'failed to execute message; message index: 0: player tried to play out of
+                                       turn'
+...
+txhash: D10BB8A706870F65F19E4DF48FB870E4B7D55AF4232AE0F6897C23466FF7871B
+```
+
+If you did not get this `raw_log`, perhaps because the transaction was sent asynchronously, use the `txhash` in the following command:
```

Comment:
```suggestion
If you did not get this `raw_log`, it might be because your transaction was sent asynchronously. You can always query a transaction by using the `txhash` by using the following command:
```